src/main/java/no/nav/tag/sykefravarsstatistikk/api/provisjonering/importering/StatistikkImportRepository.java

```diff
@@ -0,0 +1,72 @@
+package no.nav.tag.sykefravarsstatistikk.api.provisjonering.importering;
+
+import no.nav.tag.sykefravarsstatistikk.api.domene.SlettOgOpprettResultat;
+import no.nav.tag.sykefravarsstatistikk.api.domene.statistikk.SykefraværsstatistikkLand;
+import no.nav.tag.sykefravarsstatistikk.api.domene.statistikk.Sykefraværsstatistikk;
+import no.nav.tag.sykefravarsstatistikk.api.domene.statistikk.ÅrstallOgKvartal;
+import no.nav.tag.sykefravarsstatistikk.api.provisjonering.importering.integrasjon.CreateSykefraværsstatistikkFunction;
+import no.nav.tag.sykefravarsstatistikk.api.provisjonering.importering.integrasjon.DeleteSykefraværsstatistikkFunction;
+import no.nav.tag.sykefravarsstatistikk.api.provisjonering.importering.integrasjon.utils.SykefraværsstatistikkLandUtils;
+import no.nav.tag.sykefravarsstatistikk.api.provisjonering.importering.integrasjon.utils.SykefraværsstatistikkIntegrasjonUtils;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.annotation.Profile;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+import java.util.concurrent.atomic.AtomicInteger;
+
+@Profile({"local", "dev"})
+@Component
+public class StatistikkImportRepository {
+
+  private final NamedParameterJdbcTemplate namedParameterJdbcTemplate;
+
+  public StatistikkImportRepository(
+      @Qualifier("sykefravarsstatistikkJdbcTemplate")
+          NamedParameterJdbcTemplate namedParameterJdbcTemplate) {
+    this.namedParameterJdbcTemplate = namedParameterJdbcTemplate;
+  }
+
+
+    public SlettOgOpprettResultat importSykefraværsstatistikkLand(
```

Comment:
Det er et eller annet som skurrer for meg når vi har en funksjon som heter "importStatistikk", men som tar inn statistikken man skal importere som argument :P "Import" betyr for meg det å hente data fra et annet sted. Og selv om lagring av importert data er en del av importeringsprosessen, så vil jeg ikke si at denne klassen spesifikt driver med import. Heller kanskje "oppdatering" eller noe? "StatistikkOppdateringRepository"? :/ Jeg er usikker på hva som er best her. Men da kunne vi kalt denne funksjonen for "oppdaterSykefraværsstatistikkLand" - det synes jeg gir mening i alle fall. Og importStats kunne hete "oppdaterSykefraværsstatistikk".

Reply:
Will do. Bare utsetter det til etter vi har merget de forskjellige bransjene vi har.